pkgs/applications/backup/vdmfec/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "vdmfec_${version}";
```

Comment:
```suggestion
  name = "vdmfec-${version}";
```

Reply:
the filename has a _ in it, what should I do about that?

Reply:
Oh, right. You can do
```nix
stdenv.mkDerivation rec {
  pname = "vdmfec";
  version = "1.0";

  src = fetchurl {
    bla = "http://example.com/${pname}_${version}.tgz";
  };
}
```

Reply:
Hardcoding a url would not be such a bad idea in this case too because the software wasn't updated in 20 years...